<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>1000</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addColumnBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>添加分栏标记</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addSectionBtn">
        <property name="text">
         <string>添加分节标记</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteSectionBtn">
        <property name="text">
         <string>删除分节标记</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addBanBtn">
        <property name="text">
         <string>添加分栏屏蔽</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteBanBtn">
        <property name="text">
         <string>删除分栏屏蔽</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="minimumSize">
       <size>
        <width>8</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>780</width>
        <height>850</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>处理进度：</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="prevPageBtn">
        <property name="text">
         <string>上一页</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextPageBtn">
        <property name="text">
         <string>下一页</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
